### DISCUSSION TITLE
GitHub Organisation for bun

### DISCUSSION THREAD

**Discussion by lucasmichot** [oven-sh/bun#354] [Ideas]
Hey there 👋 

I believe this could be useful to have a GitHub organisation for `bun`.
This way specific repositories can be created (docs, landing page, etc...) and managed separately.

I have experience setting up orgs and am happy to help

---

cc @Jarred-Sumner

**Comment by Murtatrxx** [comment]
I’m here to help if needed 😉🥰

**Reply by Murtatrxx** [reply]
Also I got the “buncommunity” name and if devs interested I can transfer to them

**Comment by santosant** [comment]
I'd like to help with docs 🤗

**Reply by alphaolomi** [reply]
Docs have a dedicated repo at https://github.com/oven-sh/docs

**Comment by alphaolomi** [comment]
UPDATE: the project have been moved to GH org [oven-sh](https://github.com/oven-sh)
repo at https://github.com/oven-sh/bun

**Comment by Jarred-Sumner** [comment]
Done
